src/components/Game.jsx

```diff
@@ -1,37 +1,32 @@
-import React from "react"
+import React, { useState, useEffect } from "react"
 import { connect } from "react-redux"
 import { updateScore, updateRounds } from "../store/game"
 import { fetchPeople } from "../store/people"
 import Picture from "./Picture"
 
-export class Game extends React.Component {
-  constructor(props){
-    super(props)
-    this.state = {
-      selectedPerson: {},
-      renderOverlay: false,
-    }
-  }
+const Game = (props) => {
```

Comment:
Good work! using react hooks makes the component much cleaner.
A few small details to help make it even more clean:
- I really like to be specific about the things we pass a props to the component, so it's easier to read and we don't have to use `props.` throughout` the component, destructing helps a lot with that. Here in your Game component we could use it like this:

```
const Game = ({ people, roundPeople, realPerson, score, rounds, fetchPeople, updateScore, updateRound }) => {...}
```